kura/org.eclipse.kura.rest.position.provider/src/main/java/org/eclipse/kura/internal/rest/position/PositionRestService.java

```diff
@@ -84,7 +84,7 @@ public void unsetRequestHandlerRegistry(final RequestHandlerRegistry registry) {
     @Produces(MediaType.APPLICATION_JSON)
     public PositionDTO getPosition() {
         if (positionServiceImpl.isLocked()) {
-            return new PositionDTO(positionServiceImpl.getPosition());
+            return new PositionDTO(positionServiceImpl.getPosition(), positionServiceImpl.getGnssType());
```

Comment:
I know that it isn't related to this PR and it is a detail, but I'd call this getGnssType**s** or getGnssTypeSet instead of getGnssType since it returns a set not a single value.